platform/darwin/src/MGLStyleLayer.mm.ejs

```diff
@@ -5,7 +5,7 @@
   const enumProperties = locals.enumProperties;
 -%>
 // This file is generated.
-// Edit platform/darwin/scripts/generate-style-code.js, then run `make darwin-style-code`.
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code`.
```

Comment:
I think that, since the generation comes from the Darwin specific script, we should not tell developers to use `style-code` since that runs the scripts for core and all platforms. [darwin-style-code](https://github.com/mapbox/mapbox-gl-native/blob/release-ios-v3.5.0-android-v5.0.0/Makefile#L268) (most of the time) probably makes the expected changes for a developer wanting to modify the Darwin platforms implementation.